### ISSUE TITLE
Tags not detected

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#799]
Project 113 and possibly [others ](https://github.com/MotusDev/Motus-TO-DO/issues/796)have been having issues with tags not being detected on some stations while others very nearby are picking up tags. I'm not really sure what's going on here other than they are quite convinced that there is something wrong based on when/where they are deploying their tags. 

Copied from email:

> [Kennemerwind ](https://motus.org/data/receiver/deployment/plot?id=7264)7264 no tags, this cannot be true!
> [De Kooy](https://motus.org/data/receiver/deployment/plot?id=5604) 5604, no tags at all, this cannot be true! I did a manual quick check 
> And found these:
>  
> ![image](https://user-images.githubusercontent.com/55001144/92742654-d7656800-f34d-11ea-8012-31e4b9e0b4f1.png)
> 
> [Huiberts ](https://motus.org/data/receiver/deployment/plot?id=4892)4892 only tags found from 4 september on, impossible
> [Bergen ](https://motus.org/data/receiver/deployment/plot?id=5013)5013  only tags found from 8 september on, highly unlikely
> [Vinetaduin ](https://motus.org/data/receiver/deployment/plot?id=5571)5571, No tags at all, this is not true for sure, I did a manual run with tagfinder on September 2, and found 2 tags in it 52&301 so this is definably wrong.
> 
> This is just an example of detections not picked up by the system.
> My conclusions:
> •	Data is transferred to motus (as you can see GPS fixes and antenna activity is in the examples)
> •	Detection process is still seriously wrong but it looks like some receivers are treated different then others.
> •	Data before sept 4 seems to be processed differently then after sept 4 for certain receivers.
> 
> Please keep us posted on a daily basis on progress, I don’t know what I can tell the guys in the field.

In all these cases the receivers are syncing fine, there is clearly activity registering on the antennas, but no or very few tags are being identified. At the same time, they are having all kinds of issues registering tags, with errors of "no tag detected" in recording, leaving them very concerned that there is something currently not working with tag finder itself.

**Comment by StuMackenzie** [comment]
@Hussell If you want to double check this issue as it relates to #796 #784

**Comment by Hussell** [comment]
The tag registration issues were analyzed by Lucas in #800, and it seems they're due to the tags behaving outside expected parameters, so not due to an issue with the tag finder.

To help diagnose the issues with the receivers, it would be very helpful to know which tags should have been detected on each receiver. Based on past experience, this kind of thing often happens when tags haven't been registered, or tag deployments weren't updated until after the tag finder was run.

**Comment by StuMackenzie** [comment]
Yeah - I was referring to the 5 receivers listed above. You may have only rerun 4?

**Comment by leberrigan** [comment]
It looks like the rerun did not resolve the issue. Bart has re-processed the data manually and still isn't seeing the tags as expected.

In particular:
- Receiver deployment 5604 should have tags 47741, 47584, 47662, 47675
- Receiver deployment 5571 should have tags 47669 and 47760 on Sept 2 and probably other detections
- Receiver deployment 4892 should have detections from this year before Sept 4 of any tag
- Receiver deployment 5013 should have detections from this year before Sept 8 of any tag

**Comment by leberrigan** [comment]
I should note that all these receivers, except for 7264, already have detections of other tags which should rule out the receiver frequency as being the issue, right?

In addition, all tags mentioned above have been detected in the past 2 weeks on other receivers.

**Comment by Hussell** [comment]
5604: not yet rerun
5571: rerun 2020-09-26, tag 47669 showed up from 08-26 to 08-30 and tag 47760 showed up on 08-26, among ~90,000 new detections.
4892: not yet rerun
5013: rerun 2020-09-26, 1,188 new detections from 08-31 to 09-21
7264: not yet rerun

The bulk of new detections during reruns during the past few weeks has been from projects 113, 76, 123, 223, and other European projects. So far everything looks like the result of tag metadata not being entered until after the receiver data was uploaded. Every one of these receivers is connected and uploading data hourly. Reruns for 23 receivers from project 113 are in the queue, along with 3 for project 76, 5 for project 123, 6 for project 223, and 4 for other projects in Europe.

**Comment by jsayers-bsc** [comment]
I think we may have an answer to what was going on here... The collaborators here also recently emailed about the receiver at the centre of issue #784 that they were not seeing detections of tags that they had deployed near this station. Same issue as described above.

They were able to confirm the tags by using the local [find_tags program that John made](https://github.com/jbrzusto/find_tags) (which I've never used myself). But when I checked the data in R, I didn't see anything of the sort. Only one tag was showing in with good runs. and that was the one tag misidentified as a North American bird. All the rest were runs of 2.

But after the receiver was re-processed using the correct default frequency, all those other tags showed up. Upon looking into the new tags it was clear that no 166 MHz version of the tags in question were registered.

So when the pulse data were originally processed with the default frequency of 166 MHz, only the one tag ID that actually had a 166 MHz counterpart currently registered was identified. Once the frequency was corrected, tag finder now matches those tag ID's to real tags. 

I am not familiar with using the find_tags program, but I do seem to recall reading somewhere that it doesn't rely on the master database in the same way, so maybe that accounts for why it picked up tag ID that weren't in the database? 

Presumably if 166 MHz versions of the tags had been registered, but not deployed, they would have showed up in the R data as "?proj?-0#:0.0" or whatever that ID is for tags that are recognized but not associated with a deployment. But since they weren't even registered on that frequency, there were ignored entirely.
